src/main/webapp/app/quiz/participate/quiz-scoring-infostudent-modal/quiz-scoring-infostudent-modal.component.html

```diff
@@ -0,0 +1,37 @@
+<ng-template #scoringexplanation let-c="close" let-d="dismiss">
+    <div class="modal-header">
+        <h5 class="modal-title">Scoring type explanation</h5>
+        <button type="button" class="close" aria-label="Close" (click)="d()"><span aria-hidden="true">&times;</span>
+        </button>
+    </div>
+    <div class="modal-body" *ngIf="this.scoringType === 'ALL_OR_NOTHING' && this.questionType === 'mcq'" >
+        <h6>The scoring type was <span jhiTranslate="arTeMiSApp.quizExercise.scoringType.all_or_nothing"></span> This means you either get all the points for the question or non at all.</h6>
```

Comment:
Please use the enums defined in question.model.ts instead of strings for scoring type and question type. You need to declare read only constants in the component.ts file to do that. Have a look at quiz.component.ts and quiz.component.html how this works
Please use question.type and do not redefine questionType with a different string format.

Reply:
Done

Reply:
Done